servers/lambda/src/main/resources/jni-config.json

```diff
@@ -0,0 +1,46 @@
+[
```

Comment:
This one became necessary with RocksDB in Quarkus

Reply:
can you elaborate why it's necessary?

Reply:
This _file_ needs to be there for the Quarkus build

Reply:
I understand that Quarkus needs this file, I just don't understand what needs to be in this file, that's why having a comment or something would be helpful to the reader

Reply:
JSON doesn't support comments